cmd/kubeadm/app/api/env.go

```diff
@@ -0,0 +1,33 @@
+package api
+
+import (
+	"fmt"
+	"os"
+	"runtime"
+	"strings"
+)
+
+// TODO(phase2) use componentconfig
+// we need some params for testing etc, let's keep these hidden for now
+func GetEnvParams() map[string]string {
```

Comment:
Can we kill `map[string]string` and just use a struct. That way the string names only exist in one place and have only one place to be mis-typed.